tests/integration/local/invoke/test_invoke_terraform_applications.py

```diff
@@ -184,7 +184,7 @@ def test_invoke_function_with_beta_feature_expect_warning_message(self):
             "You can also enable this beta feature with 'sam local invoke --beta-features'."
         )
         self.assertNotRegex(stdout.decode("utf-8"), terraform_beta_feature_prompted_text)
-        self.assertTrue(stderr.decode("utf-8").startswith(Colored().yellow(EXPERIMENTAL_WARNING)))
+        self.assertTrue(Colored().yellow(EXPERIMENTAL_WARNING) in stderr.decode("utf-8"))
```

Comment:
We start printing some log statement in case if `samdev` is used, so the Warning message is not the first printed message any more.

https://github.com/aws/aws-sam-cli/blob/ad9a5986b434bab26159719a95ad33ba8f602d49/samcli/cli/main.py#L138-L146